src/components/Stocks.js

```diff
@@ -16,26 +18,37 @@ const Stocks = () => {
   const handleCompanyInfo = (symbol) => {
     dispatch(getCompanyInfo(symbol));
   };
+
+  const onSearchChange = (event) => {
+    dispatch(setSearchField(event.target.value));
+  };
+
+  // eslint-disable-next-line max-len
```

Comment:
Let's remove all disabled Linters, Linting helps with more cleaner and professional code 👍 

Kindly check for other occurrences 🙂